pyscriptjs/environment.yml

```diff
@@ -15,6 +15,8 @@ dependencies:
     - numpy
     - markdown
     - pip:
+          - pyodide_py==0.22
+          - micropip==0.2.2
```

Comment:
I'm confused. What are these used for?

Reply:
For py-unit tests, rather than using mock modules. Also, they include types.

Reply:
I added a comment to the environment file explaining this.